File: app/src/main/java/com/geomoby/demoapp/GeoMobyApplication.java
package com.geomoby.demoapp;

import android.content.Context;

import androidx.multidex.MultiDexApplication;

import com.geomoby.GeoMoby;

public class GeoMobyApplication extends MultiDexApplication {

    private static Context mContext;

    @Override
    public void onCreate() {
        super.onCreate();
        mContext = getApplicationContext();
    }

    public static Context getContext() {
        return mContext;
    }
}

File: app/src/main/java/com/geomoby/demoapp/GeoMobyApplication.kt
package com.geomoby.demoapp

import androidx.multidex.MultiDexApplication
import com.geomoby.demoapp.data.geomoby.GeomobyStartManager
import dagger.hilt.android.HiltAndroidApp

@HiltAndroidApp
class GeoMobyApplication : MultiDexApplication(){

    override fun onCreate() {
        super.onCreate()
        GeomobyStartManager().start(this)
    }
}